### PR TITLE
net-dns/getdns: bump version to 1.6.0_beta1

### PR DESCRIPTION
**Pull request by CaseOf**
New net-dns/getdns version has switched to cmake build system.

### CODE REVIEW

**Comment by gyakovlev** [comment]
just to clarify, maintanership was dropped here https://github.com/gentoo/gentoo/pull/14084

File: net-dns/getdns/files/stubby.initd-r2 [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/sbin/openrc-run
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License, v2 or later
+
+stubby_opts="-C ${STUBBY_CONFIG_FILE:-/etc/stubby/stubby.yml} -v ${STUBBY_LOGLEVEL:-5}"
+command="capsh"
+command_args="--inh=cap_net_bind_service --user=stubby -- -c '/usr/bin/stubby -g ${stubby_opts} >>/var/log/stubby.log 2>&1'"
+pidfile="/var/run/stubby.pid"
```
Comment (gyakovlev) [inline net-dns/getdns/files/stubby.initd-r2]: pidfile shoud go to `/run`

File: net-dns/getdns/files/stubby.initd-r2 [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+{
+	provide dns
+	need localmount net
+	after bootmisc
+	use logger
+}
+
+start_pre()
+{
+	checkpath -f -m 0644 -o stubby:stubby /var/run/stubby.pid
```
Comment (gyakovlev) [inline net-dns/getdns/files/stubby.initd-r2]: ditto, should be /run

File: net-dns/getdns/files/stubby.initd-r2 [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+	checkpath -f -m 0644 -o stubby:stubby /var/log/stubby.log
+}
+
+stop_post()
+{
+	rm /var/run/stubby.pid
```
Comment (gyakovlev) [inline net-dns/getdns/files/stubby.initd-r2]: ditto

**Comment by blueness** [comment]
psuhed

**Pull request closed without merging**